### DISCUSSION TITLE
Does Milvus support BM25 and TFIDF?

### DISCUSSION THREAD

**Discussion by songohannyc** [milvus-io/milvus#21407] [Q&A and General discussion] [answered]
Hello,

I'm new to Information Retrieval platform. I'm evaluating Milvus and OpenSearch. Does Milvus support BM25 and TFIDF at all or there is a pending Feature request? I'm unable to locate any document on those. 

Thank you for your help.

**Comment by yhmo** [comment]
No, I dont think milvus can support BM25 and TFIDF.
All the milvus index algorithm are based on vector/embedding, not based on word frequency.

**Comment by xiaofan-luan** [comment, marked as answer]
> OpenSearch

We don't support TF iDF yet, it's in longer feature plan but won't be support in the near future. So you might need to retrieve from opensearch for keyword and Milvus for vector
